### PR TITLE
add call back when focus

### PR DESCRIPTION
**Pull request by ManhNTX**

### CODE REVIEW

File: lib/src/editor.dart [hunk 1]
```diff
@@ -471,13 +477,15 @@ blockquote {
...
         },
-
         initialOptions: InAppWebViewGroupOptions(
             crossPlatform: InAppWebViewOptions(
               supportZoom: false,
               transparentBackground: true,
               useShouldOverrideUrlLoading: true,
             ),
+            ios: IOSInAppWebViewOptions(
+              disableInputAccessoryView: true,
```
Comment (dab246) [inline lib/src/editor.dart:487]: What is this used for?

File: lib/src/editor.dart [hunk 2]
```diff
@@ -753,6 +761,19 @@ blockquote {
           });
         }
       }
+    } else if (message == 'onfocus') {
+      if(!Platform.isAndroid) {
```
Comment (dab246) [inline lib/src/editor.dart:765]: Why not on Android `unfocus` it?

**Comment by dab246** [comment]
Screen Shot 2022-09-22 at 15 34 54

Why are `Signature` and `AutoComplete` here? Please rebase the latest code.

File: lib/src/editor.dart (hunk 2 above)
Comment (ManhNTX) [reply lib/src/editor.dart:765]: on Android when you add `FocusScope.of(context).unfocus();` HTML editor will lost focus too. When we remove that line of code, when the focus is on the HTML editor, the other fields will automatically remove the focus

File: lib/src/editor.dart (hunk 1 above)
Comment (ManhNTX) [reply lib/src/editor.dart:487]: disable "Done" button on ios

**Review by dab246 — APPROVED** [review]
/lgtm

**Review by hoangdat — APPROVED** [review]

**Pull request merged**